packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/fade/useFadeDiagramElements.tsx

```diff
@@ -74,9 +74,11 @@ export const useFadeDiagramElements = (): UseFadeDiagramElements => {
         elementIds: nodeId,
         fade,
       };
-      fadeElementMutation({ variables: { input } });
+      if (!readOnly) {
```

Comment:
Why? Shouldn't the palette button be inactive along with the toolbar button too?

Reply:
Done